assets/css/theme/text-styles.css

```diff
@@ -0,0 +1,17 @@
+.hero-title {
```

Comment:
this seems like component specific, shouldn't it be in like hero.css ?

Reply:
good point - i actually clarified with design prior and these are actual text styles (similar to our fe); i'll be updating them to their proper names actually (`.display` etc) but the rest of the `textStyles` + colors will **only be added later**

Reply:
okay!